examples/snapshot/employee-info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "source": {
+      "metric": "employeeCount"
+    },
+    "columns": [
+      {
+        "columnDefinition": {
+          "property": {
+            "name": "Employee.Age",
+            "qualifyingPath": "Employee"
+          }
+        }
+      },
+      {
+        "columnDefinition": {
+          "property": {
+            "name": "Employee.Start_Date",
+            "qualifyingPath": "Employee"
+          }
+        }
+      },
+      {
+        "columnDefinition": {
+          "property": {
+            "name": "Employee.Market_Compensation_Ratio",
+            "qualifyingPath": "Employee"
+          }
+        }
+      },
+      {
+        "columnDefinition": {
+          "property": {
+            "name": "Employee.Full_Name",
+            "qualifyingPath": "Employee"
+          }
+        }
+      },
+      {
+        "columnName": "Effective time",
+        "columnDefinition": {
+          "effectiveDateProperty": {}
```

Comment:
If a user comes into this file to see how to use effectiveDateProperty, is this illuminating? I probably just don't have the full picture, but I was expecting to see a date specified

Reply:
It might not be the epitome of illumination. It's a special property however that is determined by the engine for each snapshot. The overall context time range is defined in `timeIntervals` from which the engine draws which snapshots (in time) to create